packages/applicant/src/pages/dashboard/Dashboard.test.tsx

```diff
@@ -102,47 +116,82 @@ describe('Dashboard', () => {
 
   describe('should render third section', () => {
     test('should render heading', () => {
+      render(
+        <ApplicantDashboard
+          {...getProps(getDefaultProps, { hasApplications: false })}
+        />
+      );
+
       const heading = screen.getByRole('heading', {
         name: /your details/i,
       });
       expect(heading).toBeInTheDocument();
     });
 
     test('should render the 2 cards', () => {
+      render(
+        <ApplicantDashboard
+          {...getProps(getDefaultProps, { hasApplications: false })}
+        />
+      );
+
       const organisationCard = screen.getByText(
         /Change your organisation details/i
       );
       expect(organisationCard).toBeInTheDocument();
     });
   });
-});
 
-describe('migration journey feature flag', () => {
-  test('should render matching account banner', () => {
-    render(
-      <ApplicantDashboard
-        descriptionList={descriptionList}
-        hasApplications={false}
-        oneLoginMatchingAccountBannerEnabled={true}
-      />
-    );
-    const banner = screen.getByRole('heading', {
-      name: 'MATCHING ACCOUNT BANNER PLACEHOLDER',
+  describe('migration journey feature flag', () => {
+    test('Should render the error banner', () => {
```

Comment:
success and error tests in this file should include the condition under test as well eg

```suggestion
    test(`renders error banner when migrationSucceeded is 'false'`, () => {
```